client/cl_eventhandlers.lua

```diff
@@ -9,6 +10,58 @@ local function isPlayerAWitness(witnesses)
     return false
 end
 
+--hunting zones
+CreateThread(function()
+    for _, hunting in pairs(Config.Locations["hunting"]) do
+        if Config.Debug then
+            local huntingzone = CircleZone:Create(vector3(hunting.coords.x, hunting.coords.y, hunting.coords.z), hunting.radius, {
+                name = Config.Locations["hunting"].label,
+                useZ = true,
+                debugPoly = true
+            })
+
+            huntingzone:onPlayerInOut(function (isPointInside)
+                if isPointInside then
+                    QBCore.Functions.Notify("DEBUG: INSIDE HUNTING AREA: "..hunting.label, "success")
```

Comment:
Please make this into a config for DebugHuntingZone true/false or just remove all together

Reply:
Removed all togehter